packages/GA4Client/test/src/tests.js

```diff
@@ -1648,197 +1650,95 @@ describe('Google Analytics 4 Event', function () {
             window.dataLayer = undefined;
         });
 
-        // This forces the dataLayer to have a type array to make testing easier
-        // Without this, it is an array-like object of arguments
-
-        it('should standardize event names and attributes keys', function (done) {
-            mParticle.forwarder.process({
-                EventDataType: MessageType.PageEvent,
-                EventCategory: EventType.Navigation,
-                EventName: '2?Test Event ?Standardization',
-                EventAttributes: {
-                    foo: 'bar',
-                    '1?test4ever!!!': 'tester',
-                },
-            });
-
-            var expectedEventName = 'Test_Event__Standardization';
+        describe('standardizing event names and attribute keys', function () {
```

Comment:
FYI - The tests from 1653-1909 are just indented in order to be organized under a new describe block.  New tests are 1909 and down